addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -57,7 +44,6 @@ def create_alias(self):
             source.alias_id = self.with_context(alias_model_name='hr.applicant', alias_parent_model_name='hr.job').env['mail.alias'].create(vals)
             source.name = source.source_id.name
 
-
```

Comment:
Why did you remove this line ? It should stay like it was before, see: https://www.python.org/dev/peps/pep-0008/#blank-lines.